### DISCUSSION TITLE
Can't use cyrillic characters

### DISCUSSION THREAD

**Discussion by alprotasov** [nocodb/nocodb#1163] [Q&A] [answered]
Hello!
Get this error in response (Error 400)
{"msg":"insert into `nc_ztmr__design` (`Title`) values ('ыфаыв а') - Incorrect string value: '\\xD1\\x8B\\xD1\\x84\\xD0\\xB0...' for column 'Title' at row 1"}

Google said this is because of MySQL db collation.
How can I solve this problem?
PS I use docker-compose to deploy NocoDB

**Comment by wingkwong** [comment]
I couldn't reproduce this problem. Could you show us the project info like below?

Node: **v14.18.0**
Arch: **x64**
Platform: **darwin**
Docker: **false**
Database: **mysql2**
ProjectOnRootDB: **false**
RootDB: **sqlite3**
PackageVersion: **0.84.10**

**Reply by dstala** [reply]
@alprotasov 
I tried too. It worked. Looks like this is something to do with character set & collate configuration in MySQL.

On your MySQL workbench, can you execute below query & share result?
SELECT @@character_set_database, @@collation_database;

To execute query on MySQL Workbench
- Open File > New query tab
- If your database is set as default DB, execute query `SELECT @@character_set_database, @@collation_database`
- Else, execute query `SELECT DEFAULT_CHARACTER_SET_NAME, DEFAULT_COLLATION_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = 'db_name';`

In my case, I could insert 'ыфаыв а' into records with character set 'utf8mb4' && collation set to 'utf8mb4_0900_ai_ci'

**Reply by alprotasov** [reply]
Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **mysql2**
ProjectOnRootDB: **true**
RootDB: **mysql2**
PackageVersion: **0.84.10**

**Reply by alprotasov** [reply]
+----------------------------+------------------------+
| DEFAULT_CHARACTER_SET_NAME | DEFAULT_COLLATION_NAME |
+----------------------------+------------------------+
| latin1                     | latin1_swedish_ci      |
+----------------------------+------------------------+

Nice... :) switching to  utf8mb4
Thanks, guys!

**Comment by alprotasov** [comment, marked as answer]
I solved the issue for tables, but I can't solve it for comments (updated all the tables in nocodb database). How can I fix comments?

**Reply by o1lab** [reply]
Comments are stored in a different schema pointed to by ```NC_DB``` environment variable.

if you havent provided that variable - we highly suggest you to run in [production settings](https://docs.nocodb.com/getting-started/installation#production-setup). Also take a look at simple overview of [architecture](https://docs.nocodb.com/#architecture---simple-overview).

**Reply by alprotasov** [reply]
I used https://docs.nocodb.com/getting-started/installation/#docker-compose this to install it. And there is only 1 mysql instance there.

**Reply by alprotasov** [reply]
**different schema**
there are only 2 schemas there 
+--------------------+
| Database           |
+--------------------+
| information_schema |
| nocodb             |
+--------------------+
I updated nocodb and I can send Cyrillic symbols in fields. But there's a problem with comments (I mean these ones https://markuphero.com/share/PZDJjvUmveBk6NeQckCZ).

**Reply by dstala** [reply]
On the MySQL database, it might help to configure the default character set globally.
https://stackoverflow.com/a/24487309/16796268

**Reply by alprotasov** [reply]
hmmm... did ALTER TABLE nc_audit CHANGE description CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci; again and it worked. Thanks for your help again!

**Reply by o1lab** [reply]
Glad you were able to sort this out. This discussion would help others.
